### PR TITLE
Option 2: vision on about page for #1801.

### PR DESCRIPTION
**Pull request by aj-stein-nist**
# Committer Notes

This PR demonstrates approach 1 for https://github.com/usnistgov/OSCAL/issues/1801 adding the new vision to the top of the about page.

image

image

### All Submissions:

- [ ] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [ ] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [ ] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

(For reviewers: [The wiki has guidance](https://github.com/usnistgov/OSCAL/wiki/Issue-Review) on code review and overall issue review for completeness.)

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you included examples of how to use your new feature(s)?
- [ ] Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.

### CODE REVIEW

**Review by nikitawootten-nist — DISMISSED** [review]

File: docs/content/about/_index.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ aliases:
   - /about/goals/
 ---
 
+## Our Vision
+
+NIST and the community will maintain a set of OSCAL models for organizations and service providers to facilitate seamless, continuous security assessment by exchanging information about system implementation and analysis without vendor lock-in. ​
```
Comment (iMichaela) [inline docs/content/about/_index.md:20]: Suggestion:  NIST and the community will mature and maintain a set of ...

File: docs/content/about/_index.md [hunk 2]
```diff
@@ -15,6 +15,14 @@ aliases:
...
+
+Community engagement and collaboration will mature OSCAL models faster and drive the broadest international adoption possible in public and private sectors.
+
+## What is OSCAL?
+
 NIST is developing the Open Security Controls Assessment Language (OSCAL) as a standardized, data-centric framework that can be applied to an information system for documenting and assessing its security controls. Today, security controls and control baselines are represented in proprietary formats, requiring data conversion and manual effort to describe their implementation. An important goal of OSCAL is to move the security controls and control baselines from a text-based and manual approach (using word processors or spreadsheets) to a set of standardized and machine-readable formats. With systems security information represented in OSCAL, security professionals will be able to automate security assessment, auditing, and continuous monitoring processes.
```
Comment (iMichaela) [inline docs/content/about/_index.md:26]: NIST replaced 'information systems' with 'systems' in the RMF-related SPs. Suggest aligning here too.
Suggestion: The text: "to a set of standardized and machine-readable formats." should read "to a set of standardized machine-readable formats."
Text: "With systems security information represented in OSCAL, ..." should read "With systems' security information .....

**Review by iMichaela — APPROVED** [review]
Please consider fixing the two typos and the suggested text before merging.

File: docs/content/about/_index.md (hunk 2 above)
Comment (aj-stein-nist) [reply docs/content/about/_index.md:26]: Sending up a PR with these recommendations integrated.

File: docs/content/about/_index.md (hunk 1 above)
Comment (aj-stein-nist) [reply docs/content/about/_index.md:20]: Sending up a commit with these recommendations integrated.

**Comment by iMichaela** [comment]
@aj-stein-nist Let's make the "Learn More" button under  **Automated Control-Based Assessment** Supporting Control-Based Risk Management with Standardized Formats read "OSCAL Vision". The button links to the About page already, which, when updated will start with the vision.

**Comment by iMichaela** [comment]
> I am not sure how we can more strongly hint from the landing page without making a large, unwiedly button by switching `Learn More` to `Learn about OSCAL and our vision` (for a button, that gets wordy with unclear benefit. I will merge this for now and reopen accordingly if it comes up during sprint review.

To answer your question above @aj-stein-nist - Yes, "Learn about OSCAL and our vision" is too long. About and Learn tabs are merged so the front page will have an "About" if people do not care to click on the "OSCAL Vision" button (if we rename it)

**Comment by aj-stein-nist** [comment]
> > I am not sure how we can more strongly hint from the landing page without making a large, unwiedly button by switching `Learn More` to `Learn about OSCAL and our vision` (for a button, that gets wordy with unclear benefit. I will merge this for now and reopen accordingly if it comes up during sprint review.
> 
> To answer your question above @aj-stein-nist - Yes, "Learn about OSCAL and our vision" is too long. About and Learn tabs are merged so the front page will have an "About" if people do not care to click on the "OSCAL Vision" button (if we rename it)

We can discuss during review, but with the exception of "OK", I had learned over the last few years that buttons should connote some form of action, and [the USWDS implies this](https://designsystem.digital.gov/components/button/#guidance). I have been scolded before for poor UX around these things in other projects prior so this is why I wanted to leave it as is.

**Pull request merged**